cli/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "xstate-cli",
+  "version": "0.0.1",
+  "bin": {
+    "xstate": "./bin/index.js"
+  },
+  "scripts": {
+    "prepare": "esbuild src/bin.ts --outfile=bin/index.js --bundle --platform=node --target=node14 --external:fsevents"
```

Comment:
is `fsevents` the only external that we want to keep as external?

Reply:
Probably not, currently the bundle is like 17MB so something's going wrong. Probably an issue with preconstruct/dev and this?

Reply:
i might be wrong but it sounds like you are bundling the whole thing, while you might only want to bundle what's inside your source files, leaving all of the deps and peer deps as external

Reply:
Yess, that makes sense

Reply:
Fixed, now down to 4.8kb.